pandas/core/generic.py

```diff
@@ -2189,6 +2189,9 @@ def to_excel(
         Once a workbook has been saved it is not possible to write further
         data without rewriting the whole workbook.
 
+        If you'd like to be more flexible with the output formatting,
```

Comment:
As mentioned in https://github.com/pandas-dev/pandas/issues/47525#issuecomment-1169199316=, you can update the ecosystem docs in `doc/source/ecosystem.rst` to mention StyleFrame and then use a Sphix reference to mention there are ecosystem packages to help with output formatting

Reply:
Cool, tahnks. I added the package to the ecosystem doc and used a Sphinx reference in the docstring.

Reply:
Looks like `StyleFrame` needs to be referenced in that doc somewhere